source/includes/fact-filter-queries.rst

```diff
@@ -0,0 +1,11 @@
+Allows you to filter and limit the amount of entries displayed in 
+the :guilabel:`Query Converter` pane.
+
+
+.. procedure::
+   :style: normal
+
+   .. step:: Select your query
+      Enter a text string to match against the object name and SQL 
+      syntax in the :guilabel:`Filter` text box or click  
+      :icon-fa5:`ellipsis-v` and select from the supplied filters.
```

Comment:
"supplied filters" seems overly vague, maybe something like "click the ellipsis to filter queries by their conversion status".